### ISSUE TITLE
Show enemy as plural in narrative text is more than one

### ISSUE THREAD

**Issue by maxim-zhao** [maxim-zhao/psrp#10]
The text like “you defeated the <enemy>” should pluralise to reflect the intended plural number of enemies. Maybe also show the number in the name box? (Decreasing when dead, hidden when starting at 1, e.g. Dark Force, LaShiec.)

**Comment by maxim-zhao** [comment]
Tricky to deal with pluralisation...
- Ice Man => Ice Men
- Living Dead => Living Dead
- Marshes => Marshes
- Gold Lens => Gold Lenses
I guess we'd need a whole plural table...

**Comment by LeonisX** [comment]
Yes, we need a separate table, in Russian there are even more rules for ending words.
